src/SelectLocation.js

```diff
@@ -66,10 +67,21 @@ export default class SelectLocation extends Component {
       geocodeAddressString: location.address,
       formValue: props.value || "",
       lat: "Loading...",
-      lng: "Loading..."
+      lng: "Loading...",
+      style: "streets-v9"
```

Comment:
I guess this is actually the right place to set the default

Reply:
✅ Map now loads satellite layer on render.